dbt_server/state.py

```diff
@@ -57,10 +87,15 @@ def reset(self):
             self.parser = None
 
 
+# Global singleton manifest cache object.
 LAST_PARSED = CachedManifest()
 
 
 class StateController(object):
+    """StateController stores all dbt core required state including project
+    metadata and manifest, provides core interface to caller. The core will
+    process according to the state."""
```

Comment:
```suggestion
    """StateController stores all dbt core required state including project
    metadata and manifest, and provides core interface to caller. Any core
    invocations will process according to the current state."""
```
If this isn't what you were going for, feel free to ignore!